convertGRIPtoJSON.py

```diff
@@ -0,0 +1,109 @@
+import json
+from collections import OrderedDict
+from bs4.element import NavigableString  # bs4.element.NavigableString
+from bs4 import BeautifulSoup
+
+
+def iter_values(value, returned_list: list) -> list:
```

Comment:
Bad function naming ( I had to ;) ) and bad argument naming (returned_list -> you don't return it)